pep/app/pods/components/page/sidebar/widgets/favorites/component.ts

```diff
@@ -36,7 +55,7 @@ export default class PageSidebarWidgetsFavorites extends Component<PageSidebarWi
      */
     @action
     onElementChange() {
-        this.loadFromUserPreferences();
+        taskFor(this.loadFromUserPreferences).perform();
```

Comment:
This also should have some logic so that it only refetches the documents if the array of document ids changed, not just anytime prefs change, since we wouldnt want this to reload if the user changed their Theme for example. 

This would possibly be less necessary if we had the smart cache usage, tho probably still good to have, but maybe it can just be a `TODO` comment for now?